lxd/util/http.go

```diff
@@ -267,7 +267,7 @@ func ListenAddresses(configListenAddress string) ([]string, error) {
 	listenIP := net.ParseIP(unwrappedConfigListenAddress)
 	if listenIP != nil || !strings.Contains(unwrappedConfigListenAddress, ":") {
 		// Use net.JoinHostPort so that IPv6 addresses are correctly wrapped ready for parsing below.
-		configListenAddress = net.JoinHostPort(unwrappedConfigListenAddress, fmt.Sprintf("%d", shared.HTTPSDefaultPort))
+		configListenAddress = net.JoinHostPort(unwrappedConfigListenAddress, fmt.Sprint(shared.HTTPSDefaultPort))
```

Comment:
note for the future, lets prefer using strconv.Itoa or Format rather than fmt.Sprint in the future to avoid any unexpected formatting bugs.

Reply:
Fixed for the few occurrences in that PR.